.github/actions/upgrade-testing/action.yml

```diff
@@ -46,7 +46,7 @@ runs:
         CLIENT_SKIP_UPGRADE: "true"
         CLIENT_START_PROCESS: "false"
         BINARY_NAME_SUFFIX: "ubuntu-22-amd64"
-        UPGRADES_SLEEP_TIME: 220
+        UPGRADES_SLEEP_TIME: 300
```

Comment:
@gzukel  we should use block height here instead of a fixed time, 
Something like this 
https://github.com/zeta-chain/zeta-node/blob/189eb6e84ad8b3a323909b321c64c4bce88592a1/testutil/network/network_setup.go#L552-L563